test/role_test.js

```diff
@@ -138,6 +139,52 @@ helper.secrets.mockSuite(helper.suiteName(__filename), ['app', 'azure'], functio
     assert(roles.some(role => role.roleId === 'thing-id:' + clientId));
   });
 
+  test('listRoleIds', async () => {
+    await helper.Roles.modify((roles) => roles.splice(0));
+
+    // Create 4 dummy roles
+    await helper.apiClient.createRole('thing-id:' + clientId, {
+      description: 'test role',
+      scopes: ['dummy-scope-1', 'auth:create-role:*', 'dummy-scope-2'],
+    });
+    for (let i=0;i<3;i++) {
+      let tempClientId = clientId + i;
```

Comment:
Given that we are dealing with roles per say, should we rename this to `tempRoleId`?

Reply:
Right, that makes more sense